test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/address/SubscribedChangeAddressController.scala

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
File name needs to be corrected : SubscribedChangeAddressControllerSpec.scala

Reply:
whoops, I@ll change that

Reply:
fixed